backend/ecs_tasks/delete_files/main.py

```diff
@@ -98,10 +108,9 @@ def execute(queue_url, message_body, receipt_handle):
         with s3.open(object_path, "rb") as f:
             source_version = f.version_id
             logger.info("Using object version %s as source", source_version)
-            infile = load_parquet(f)
             # Write new file in-memory
-            logger.info("Generating new parquet file without matches")
-            out_sink, stats = delete_matches_from_file(infile, cols)
+            compressed = object_path.endswith(".gz")
```

Comment:
we should mention in the docs that we determine compression based on file extension (similar to the Athena docs https://docs.aws.amazon.com/athena/latest/ug/compression-formats.html)